### PR TITLE
Handle events in main screen

### PR DESCRIPTION
**Pull request by thaohq-0814**
Handle Events in Main Screen
Link: https://edu-redmine.framgia.vn/issues/5383

### CODE REVIEW

File: app/src/main/java/com/framgia/music_50/screen/main/MainActivity.java [hunk 1]
```diff
@@ -26,4 +58,44 @@ public void onSuccess(List<Track> data) {
...
+                    fragment = new SearchFragment();
+                }
+                mNavigator.goNextChildFragment(getSupportFragmentManager(), R.id.layoutContainer,
+                        fragment, true, SearchFragment.TAG);
+                return true;
+            case R.id.actionLibrary:
+                fragment = getSupportFragmentManager().findFragmentByTag(LibraryFragment.TAG);
+                if (fragment == null) {
+                    fragment = new LibraryFragment();
+                }
```
Comment (buivanhieu7112) [inline app/src/main/java/com/framgia/music_50/screen/main/MainActivity.java]: lặp code, tách method ra bạn

File: app/src/main/java/com/framgia/music_50/screen/main/MainActivity.java [hunk 2]
```diff
@@ -26,4 +58,44 @@ public void onSuccess(List<Track> data) {
...
     }
+
+    @Override
+    public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
+        Fragment fragment;
+        switch (menuItem.getItemId()) {
+            case R.id.actionHome:
+                fragment = getSupportFragmentManager().findFragmentByTag(HomeFragment.TAG);
+                if (fragment == null) {
+                    fragment = new HomeFragment();
```
Comment (buivanhieu7112) [inline app/src/main/java/com/framgia/music_50/screen/main/MainActivity.java]: check conversion 2.11

File: app/src/main/java/com/framgia/music_50/screen/main/MainActivity.java [hunk 3]
```diff
@@ -26,4 +58,44 @@ public void onSuccess(List<Track> data) {
...
+                }
+                mNavigator.goNextChildFragment(getSupportFragmentManager(), R.id.layoutContainer,
+                        fragment, true, HomeFragment.TAG);
+                return true;
+            case R.id.actionAudio:
+                fragment = getSupportFragmentManager().findFragmentByTag(AudioFragment.TAG);
+                if (fragment == null) {
+                    fragment = new AudioFragment();
```
Comment (buivanhieu7112) [inline app/src/main/java/com/framgia/music_50/screen/main/MainActivity.java]: as above

File: app/src/main/java/com/framgia/music_50/screen/main/MainActivity.java [hunk 4]
```diff
@@ -26,4 +58,44 @@ public void onSuccess(List<Track> data) {
...
+                }
+                mNavigator.goNextChildFragment(getSupportFragmentManager(), R.id.layoutContainer,
+                        fragment, true, AudioFragment.TAG);
+                return true;
+            case R.id.actionSearch:
+                fragment = getSupportFragmentManager().findFragmentByTag(SearchFragment.TAG);
+                if (fragment == null) {
+                    fragment = new SearchFragment();
```
Comment (buivanhieu7112) [inline app/src/main/java/com/framgia/music_50/screen/main/MainActivity.java]: as above

**Review by buivanhieu7112 — APPROVED** [review]

**Comment by thaohq-0814** [comment]
ready

File: app/src/main/java/com/framgia/music_50/screen/home/HomeFragment.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.framgia.music_50.R;
+
+public class HomeFragment extends Fragment {
+    public static final String TAG = HomeFragment.class.getSimpleName();
+
+    public HomeFragment() {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_50/screen/home/HomeFragment.java]: remove constructer

File: app/src/main/java/com/framgia/music_50/screen/audio/AudioFragment.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.framgia.music_50.R;
+
+public class AudioFragment extends Fragment {
+    public static final String TAG = AudioFragment.class.getSimpleName();
+
+    public AudioFragment() {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_50/screen/audio/AudioFragment.java]: remove constructer

File: app/src/main/java/com/framgia/music_50/screen/library/LibraryFragment.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.framgia.music_50.R;
+import com.framgia.music_50.screen.audio.AudioFragment;
+
+public class LibraryFragment extends Fragment {
+    public static final String TAG = LibraryFragment.class.getSimpleName();
+
+    public LibraryFragment() {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_50/screen/library/LibraryFragment.java]: remove constructer

File: app/src/main/java/com/framgia/music_50/screen/search/SearchFragment.java [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.framgia.music_50.R;
+import com.framgia.music_50.screen.audio.AudioFragment;
+
+public class SearchFragment extends Fragment {
+    public static final String TAG = SearchFragment.class.getSimpleName();
+
+    public SearchFragment() {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_50/screen/search/SearchFragment.java]: remove constructer fragment

**Comment by thaohq-0814** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**